src/main/java/org/opensha/commons/logicTree/JsonAdapterHelper.java

```diff
@@ -0,0 +1,111 @@
+package org.opensha.commons.logicTree;
```

Comment:
This is a useful utility class that we might promote to a more general location and reuse, but here is fine for now